### ISSUE TITLE
code for open source？

### ISSUE THREAD

**Issue by hplegend** [Z-Pengcheng/MGS-SLAM#1]
Hi，is there possibility for open the source code？

**Comment by Z-Pengcheng** [comment]
of course, it will be open source if it is accepted. Now, our paper is under review. Thank for your waiting.

**Comment by Vincent630** [comment]
> of course, it will be open source if it is accepted. Now, our paper is under review. Thank for your waiting.
is there any plan to open the source code ?  look forward your great work
